.github/styles/vocab.txt

```diff
@@ -221,6 +221,7 @@ productional
 Protobuf
 proxying
 Proxying
+pseudononymized
```

Comment:
```suggestion
pseudonymized
```

Right? 😅

Reply:
I was today years old when I learned this is the correct way to spell/say this.  I guess my brain always thought...  `a`-`nonym`-`ize` perhaps, "to make unnamed" or something?.  ...And therefore, `psuedo`-`nonym`-`ize`, "to kinda make unnamed" or something.  How wrong I was!